src/github/folderRepositoryManager.ts

```diff
@@ -1800,25 +1800,18 @@ export class FolderRepositoryManager implements vscode.Disposable {
 			return null;
 		}
 
-		const headGitHubRepo = this.gitHubRepositories.find(
-			repo => repo.remote.remoteName === this.repository.state.HEAD?.upstream?.remote,
-		);
-
-		// Find the github repo that matches the upstream
+		// Search through each github repo to see if it has a PR with this head branch.
 		for (const repo of this.gitHubRepositories) {
-			if (repo.remote.remoteName === this.repository.state.HEAD.upstream.remote) {
-				const matchingPullRequest = await repo.getPullRequestForBranch(
-					`${headGitHubRepo?.remote.owner}:${this.repository.state.HEAD.upstream.name}`,
-				);
-				if (matchingPullRequest && matchingPullRequest.length > 0) {
-					return {
-						owner: repo.remote.owner,
-						repositoryName: repo.remote.repositoryName,
-						prNumber: matchingPullRequest[0].number,
-						model: matchingPullRequest[0],
-					};
-				}
-				break;
+			const matchingPullRequest = await repo.getPullRequestForBranch(
+				this.repository.state.HEAD.upstream.name,
```

Comment:
I'm probably mistaken since its new code to me, but are we losing the `<fork_name>:<branch_name>` that is necessary when a fork PRs into an upstream remote?

Reply:
You are correct the `<fork_name>:<branch_name>` is gone. However, the GraphQL GH API doesn't seem to need it. If you're curious, I used the [GitHub GraphQL explorer](https://docs.github.com/en/graphql/overview/explorer) to figure this out. The query I used to get https://github.com/microsoft/vscode-pull-request-github/pull/3509 was:
```
query { 
  repository(owner:"microsoft", name:"vscode-pull-request-github") {
    pullRequests(first:1, headRefName:"alexr00/testPRFromFork") {
      nodes {
        title
      }
    }
  }
}
```
The replacing the `headRefName` with `<fork_name>:<branch_name>` string causes no PR to be returned.